src/pages/home/report/ReportActionCompose/SuggestionMention.tsx

```diff
@@ -270,9 +298,11 @@ function SuggestionMention(
                 }
 
                 return true;
-            });
+            }) as Array<PersonalDetails & {weight: number}>;
+
+            // at this point we are sure that the details are not null, since empty user details have been filtered in the previous step
```

Comment:
```suggestion
            // At this point we are sure that the details are not null, since empty user details have been filtered in the previous step
```